app/templates/search/_opportunity_data.html

```diff
@@ -0,0 +1,18 @@
+<br />
```

Comment:
Rather than the `<br />` here it might be better to add some `padding-top` styling to the new `search-page-opportunity-data` class of the section that wraps this (in `application.scss`).

Reply:
Are you ok with explaining why it's preferable for my benefit? Also, would that also need to be done in frontend-toolkit?

Reply:
Someone more frontend-y might be able to explain better, but I think the argument is that HTML markup should always be semantic - i.e. relate to the logical structure of the page, and not just how it's displayed - so `<br>` is used when a line needs to break in the middle of a paragraph - like when displaying an address, or poetry, something like that.  Spacing between paragraphs or sections on the page is a purely visual thing - there is no "meaning" to this space other than the way it should look on a particular display.  So in that case it is preferable to use CSS styles.

This is partly to do with accessibility - for example, screen readers do their best to interpret  HTML structure on the page but will (I think) generally ignore CSS styles.  So it's best to keep the HTML structure as neat and logical as possible, rather than clutter it with elements that are only there for page layout.

Reply:
That was a good explanation, thanks a lot, I discussed it with Sam and Laurence and will add bottom padding to filter section wrapper class "search-page-filters".

Reply:
Done now :)